EnhancedMediaPlayer/MediaPlayerView.swift

```diff
@@ -92,6 +92,34 @@ public struct MediaPlayerView: View {
             .tint(color)
             .wrapInButton(action: { viewModel.onTapAction?(control) })
     }
+
+    private func timeFormat(_ timeInSeconds: Float64) -> String {
```

Comment:
No, that's not how you do it.

There's a class called `DateComponentsFormatter`. Take this as an example:

```swift
import Foundation

let totalSeconds: Float64 = 1000

let formatter = DateComponentsFormatter()
// Here you can check if `totalSeconds` is > 3600 or not to see if you want to add `.seconds`
formatter.allowedUnits = [.hour, .minute, .second]
formatter.unitsStyle = .positional
formatter.zeroFormattingBehavior = [.pad]

let formattedString = formatter.string(from: TimeInterval(totalSeconds))

print(formattedString!) // "00:16:40"
```

Reply:
Ohh nice!